packages/analytics/dashboard-renderer/README.md

```diff
@@ -0,0 +1,68 @@
+# @kong-ui-public/dashboard-renderer
+
+Render Analytics charts on a page from a JSON definition.
+
+- [Requirements](#requirements)
+- [Usage](#usage)
+  - [Install](#install)
+  - [Props](#props)
+
+## Requirements
+
+- `vue` must be initialized in the host application
+- A plugin providing a `QueryBridge` must be installed in the root of the application.
```

Comment:
suggestion: can this be better documented, explained? (I haven't read the rest yet)

Reply:
I fleshed out the documentation a bit more.  We haven't yet written a "real" query bridge yet; it's something that needs to live in the host app since (for example) in Konnect it is in charge of taking a query and passing it to the appropriate method in the Konnect SDK.